src/manifold/src/manifold.cpp

```diff
@@ -260,7 +260,10 @@ MeshGL Manifold::GetMeshGL(glm::ivec3 normalIdx) const {
       out.triVerts[3 * tri + i] = impl.halfedge_[3 * oldTri + i].startVert;
 
     if (meshID != lastID) {
-      addRun(out, runNormalTransform, tri, meshIDtransform.at(meshID));
+      Impl::Relation rel;
+      auto it = meshIDtransform.find(meshID);
+      if (it != meshIDtransform.end()) rel = it->second;
```

Comment:
Is there a case where this is expected, or should this be an assert?

Reply:
I am not entirely sure, it seems that the our python test cases would somehow trigger this.

Reply:
That seems worth investigating. I think it implies we have marked some faces as coming from an ID for which there is no transform. Those things should be 1:1. We definitely don't have much testing of this in the Python bindings, so we could have a bug there.

Reply:
indeed, I think I saw this happening previously as well

Reply:
maybe we can make an issue for this and merge this first?

Reply:
Sounds good.